src/main/java/com/hierynomus/protocol/Packet.java

```diff
@@ -19,6 +19,8 @@
 
 public interface Packet<P extends Packet<P, T>, T extends Buffer<T>> {
 
+    String UNI_ENCODING = "UTF-16LE";
```

Comment:
The intent of the `protocol` package is to split it off and share it with the `sshj` library which I also maintain. The UTF-16LE encoding is very SMB/NTLM specific, so this is rather the wrong location for this constant.